package.json

```diff
@@ -66,7 +68,7 @@
     "build:examples": "node scripts/buildexamples.js",
     "build:src": "tsc --project src && node scripts/copyfiles.js",
     "build:test": "webpack --config test/webpack.conf.js",
-    "build:serverextension": "cd jupyterlab && npm run build && cd ..",
+    "build:serverextension": "cd jupyterlab && webpack",
```

Comment:
I just checked on OSX and Windows and the `cd ..` isn't required for any of them, so I removed the other uses in e0acbd6.